packages/components/ContextualMenu/src/ContextualMenu.tsx

```diff
@@ -83,10 +84,15 @@ export const ContextualMenu = compose<ContextualMenuType>({
     if (renderData.state == undefined) {
       return null;
     }
+
     return (
       <CMContext.Provider value={renderData.state.context}>
         <Slots.root>
-          <Slots.container>{children}</Slots.container>
+          <Slots.container>
+            <ScrollView contentContainerStyle={{ flexDirection: 'column', flexGrow: 1}} showsVerticalScrollIndicator={true}>
+              {children}
+            </ScrollView>
+          </Slots.container>
```

Comment:
Any reason why we couldn't have just exposed the prop on `container` and let the developer decide if they wanna add a ScrollView as a child or not?

Reply:
That's a good question. I think this is a discussion we need to have, as to how much we want to leave to users to decide? The current implementation just requires the users to decide how tall they want a menu to be. If their items list gets too long for the height fixed menu, the scrollView will be added. What you are suggesting is, they can choose both the height of the menu AND whether they want to add a ScrollView at all in any case. 

I think Lineage folks need a quick fix so I'm implementing this without official redlines actually. We should table this for sure, and we should have this design discussion eventually.

Reply:
Yeah, nothing _too_ against the implementation. The thought initially came up because macOS menus by default don't have scrollviews. macOS has different scrolling behavior for overflowing menus, but it's really custom such that I don't know I could implement it in RN to begin with and will probably just resort to a scrollview 😅.

This is definitely something that could be left for, say, V1 Callout/ContextualMenu =)